employee-rostering-frontend/package.json

```diff
@@ -36,6 +36,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "typecheck": "npx tsc --noEmit",
```

Comment:
@Christopher-Chianelli Sorry for the confusion, I was wrong here. Npx manual:

> ```
> npx [options] <command>[@version] [command-arg]...
>
> Executes <command> either from a local node_modules/.bin, or from a central
> cache, installing any packages needed in order for <command> to run.
>
> By default, npx will check whether <command> exists in $PATH, or in the local
> project binaries, and execute that.
>```

So if you have `npx tsc` it will first look for `tsc` on `$PATH` and then in `./node_modules/.bin/` which it will find:

```sh
$ file ./node_modules/.bin/tsc 
./node_modules/.bin/tsc: symbolic link to ../typescript/bin/tsc
```

So `npx tsc` is actually same as `./node_modules/typescript/bin/tsc` and the deprecated `tsc` package isn't anyhow involved in this.

Reply:
oddly enough, `npx typescript` also works (but it installs something). Will revert.